components/ConfigFetch.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import { useEffect } from 'react';
+import { edgeConfigStore } from '../src/stores/edge-config-store';
+
+const ConfigFetch = () => {
```

Comment:
```suggestion
const configFetch = () => {
```
Not a react component, can follow function snakeCase trend

Reply:
![image](https://github.com/user-attachments/assets/1a71f66d-fad1-4488-a0d0-2c2c4d402dae)

I actually tried that before and it yelled at me.